src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/calendar/BankHolidaysTest.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.calendar;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.net.URL;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.hmcts.reform.wataskmanagementapi.services.calendar.DateCalculator.DEFAULT_NON_WORKING_CALENDAR;
+
+public class BankHolidaysTest {
+
+    @Test
+    void should_create_full_object_and_checkDates() throws IOException {
```

Comment:
method name should be change to this `should_create_full_object_and_check_dates`